src/main/java/net/imglib2/algorithm/convolution/fast_gauss/FastGauss.java

```diff
@@ -88,11 +88,23 @@ public class FastGauss
 		return new LineConvolution<>( new FastGaussConvolverRealType( sigma ), direction );
 	}
 
+	/**
+	 * @param sigmas the standard deviations of the Gaussian blur (in each dimension)
+	 * @param input the input data to blur
+	 * @param output the preallocated output buffer
+	 * @implNote op name='filter.gauss', type=Computer
```

Comment:
Hmm, interesting. We may want to make a post on Image.sc asking for people's suggestions on what Ops should do for naming w.r.t. approximate vs. exact algorithms like this. Is it OK for `filter.gauss` to be approximate by default? Should the exact and approximate versions have different names? Or can we use a hint somehow?

Reply:
* As we talked about Op naming, the fundamental idea is that numerical equivalence is not required for two Ops to share a name. So I think that both should be named `filter.gauss`. I don't like any differentiation between the names because then you're being explicit about implementation details, contrary to the Ops design.
* Without additional changes, `filter.gauss` won't be approximate by default but would be approximate *always*, which is probably bad. Actually, right now `filter.gauss(sigma, source, target)` won't work because there's a conflict, and one *will* have to be prioritized. I'd lean towards prioritizing the approximate one for speed, and we should likely provide a hint that could be used to avoid this Op. There are likely additional scenarios where this would be useful, such as convolution in SciJava Ops Image.

As is, we need to prioritize one of the Ops before we merge this PR, and we may want to implement the Precision Hint upstream beforehand as well.

Reply:
I like the idra of a precision hint. It is more general than only this Op. E.g. arbitrary precision arithmetic.